externals/kyuubi-flink-sql-engine/src/test/scala/org/apache/kyuubi/engine/flink/operation/FlinkOperationSuite.scala

```diff
@@ -349,6 +347,8 @@ class FlinkOperationSuite extends WithFlinkSQLEngine with HiveJDBCTestHelper {
     withJdbcStatement() { statement =>
       val metaData = statement.getConnection.getMetaData
       var resultSet = metaData.getSchemas(null, null)
+      val DEFAULT_BUILTIN_CATALOG = "default_catalog"
```

Comment:
How about `TableConfigOptions.TABLE_CATALOG_NAME.defaultValue()` instead?

Reply:
good idea. I agree with you.